src/views/AlbumContent.vue

```diff
@@ -157,71 +152,80 @@
 </template>
 
 <script>
+// eslint-disable-next-line node/no-extraneous-import
+import { addNewFileMenuEntry } from '@nextcloud/files'
+import { generateRemoteUrl } from '@nextcloud/router'
+import { getCurrentUser } from '@nextcloud/auth'
 import { mapActions, mapGetters } from 'vuex'
+import { NcActions, NcActionButton, NcButton, NcModal, NcEmptyContent, NcActionSeparator, NcLoadingIcon, isMobile } from '@nextcloud/vue'
+import { Upload, UploadPicker } from '@nextcloud/upload'
+import debounce from 'debounce'
 
-import MapMarker from 'vue-material-design-icons/MapMarker'
-import ShareVariant from 'vue-material-design-icons/ShareVariant'
-import Plus from 'vue-material-design-icons/Plus'
-import Pencil from 'vue-material-design-icons/Pencil'
-import Delete from 'vue-material-design-icons/Delete'
-import ImagePlus from 'vue-material-design-icons/ImagePlus'
 import Close from 'vue-material-design-icons/Close'
+import Delete from 'vue-material-design-icons/Delete'
 import Download from 'vue-material-design-icons/Download'
 import DownloadMultiple from 'vue-material-design-icons/DownloadMultiple'
+import ImagePlus from 'vue-material-design-icons/ImagePlus'
+import MapMarker from 'vue-material-design-icons/MapMarker'
+import Pencil from 'vue-material-design-icons/Pencil'
+import Plus from 'vue-material-design-icons/Plus'
+import PlusSvg from '@mdi/svg/svg/plus.svg'
```

Comment:
Any reason for reordering ? Maybe we should align to limit those kind of changes

Reply:
Cleaning messy code :)